.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/automata-dcap-attestation"]
```

Comment:
I don't know if this will cause any issues but currently the repo uses node to install the contract dependencies.

Reply:
I installed import '@nomicfoundation/hardhat-foundry' (https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-foundry) It will help us use both `yarn build:all` and `forge build`. This is needed because automata project uses foundry